### PR TITLE
fix(riscv): add set_admin for riscv and fix contract_call missing bech32 address

### PR DESCRIPTION
**Pull request by LycrusHamster**
**What type of PR is this?**

fix(riscv): add set_admin for riscv
fix(riscv): fix contract_call missing bech32 address

**What this PR does / why we need it**:

missing set_admin for riscv is a defect.
the bech32 address is not supported in riscv and now it has been added, BUT IT IS NOT TESTED!!

**Special notes for your reviewer**:

Java SDK: done
Js SDK: pending

and the contract is lack of read and write permission control which should be fixed in future.
and the test cases, both unit and e2e, are break down, which should be fixed in future.
and methods of services not is registered statically/hard coded into the riscv, which leads huobi-chain and huobi-chain-payment have to diverse code, It is a must to find a way to deal with this!

### CODE REVIEW

File: services/riscv/src/authorization.rs [hunk 1]
```diff
@@ -13,7 +13,9 @@ pub enum Kind {
 
 pub struct Authorization {
     enabled:       Box<dyn StoreBool>,
+    // deployer ->
```
Comment (KaoImin) [inline services/riscv/src/authorization.rs:16]: Missing sth

**Review by KaoImin — APPROVED** [review]

File: services/riscv/src/lib.rs [hunk 2]
```diff
@@ -5,6 +5,9 @@ pub mod error;
 pub mod types;
 pub mod vm;
 
+// #[cfg(test)]
+// mod tests;
```
Comment (KaoImin) [inline services/riscv/src/lib.rs:9]: delete this

**Review by KaoImin — APPROVED** [review]

File: services/riscv/src/lib.rs (hunk 2 above)
Comment (LycrusHamster) [reply services/riscv/src/lib.rs]: removed

File: services/riscv/src/authorization.rs (hunk 1 above)
Comment (LycrusHamster) [reply services/riscv/src/authorization.rs]: fixed

**Pull request merged**